roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
@brenton in that case, I don't see an issue with assuming OCP if using RHEL as an interim workaround.